test/ssl_ctx_test.c

```diff
@@ -34,8 +76,30 @@ static int test_set_min_max_version(int idx_tst)
     SSL *ssl = NULL;
     int testresult = 0;
     version_test t = version_testdata[idx_tst];
+    const SSL_METHOD *meth = NULL;
```

Comment:
Doesn't this tend to indicate to you that we need a supported API that does the equivalent of this testing logic ...